src/doc-printer.js

```diff
@@ -12,8 +12,8 @@ function rootIndent() {
   return {
```

Comment:
Could you explain why these changes were required?

Reply:
Use for `blockquote` (leading `> `)

Reply:
https://github.com/prettier/prettier/pull/2943/files#r142024814